UPDATING.md

```diff
@@ -2089,6 +2089,10 @@ Remove unnecessary parameter ``open`` in PostgresHook function ``copy_expert`` f
 
 Change parameter name from ``visibleTo`` to ``visible_to`` in OpsgenieAlertOperator for pylint compatible
 
+`OpsgenieAlertHook` constructor does not take additional arguments or keyword arguments anymore.
+Changed the return type of `OpsgenieAlertHook.get_conn` to return a `opsgenie_sdk.AlertApi` object instead of a `requests.Session` object.
+Changed the return type of `OpsgenieAlertHook.execute` to return a `opsgenie_sdk.SuccessResponse` object instead of a `Any` type.
+
```

Comment:
This section is Updating of Airflow 2.0 - this is not relevant here.
Users who switch to 2.0 has no action item as the provider we are now upgrading is not compatible with Airflow 2.

These entries means that this PR is a breaking change (which we welcome!)
@potiuk since you generate the change log automatically I wonder what is the best way to notify the users?
Should we ask to move these records to 
https://github.com/apache/airflow/blob/main/airflow/providers/opsgenie/CHANGELOG.rst
with 3.0.0 ?
Alternatively we can maybe put this information in the PR description or/and in the commit message and the change log would refer to it?

Reply:
Yeah. Moving to Opsgenie changelog at the top is best. Not all of it is generated automatically, only list of issues so you can move the comments to the top of the changelog - and I will add the right version after.

Reply:
As requested I moved this section to the top of `/airflow/providers/opsgenie/CHANGELOG.rst`. I have put that in a [breaking change section](https://github.com/apache/airflow/pull/20263/files#diff-58c14e916c12a35f7fa03cd5e28ce661151c914c0786dfac7c3c0a1a8b96aa69R22) for now.

@potiuk  let me know if this looks good to you.